packages/app/src/routers/SavedviewsRouter/components/ColorOptionComponents.tsx

```diff
@@ -0,0 +1,212 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *
+ * This code is for demonstration purposes and should not be considered production ready.
+ *--------------------------------------------------------------------------------------------*/
+
+// tslint:disable:@typescript-eslint/no-empty-function
```

Comment:
We're not using tslint, but dont disable linting rules :)

Reply:
deleted that line